### PR TITLE
feat: store kv && prefill attn support padding.

### PR DESCRIPTION
**Pull request by wwens7**

### CODE REVIEW

File: mojo_opset/backends/ttx/kernels/ilu/kv_cache.py [hunk 1]
```diff
@@ -259,21 +273,17 @@ def store_paged_kv_impl(
...
-    n = chunk_indices.shape[0]
-    if n == 0:
-        return key_cache, value_cache
+    batch_size: int = block_table.shape[0]
+    max_chunks_per_seq = block_table.shape[1]
 
-    # One chunk per program; grid[0] = ceil(n / BLOCK)
-    BLOCK = 1
-    grid = (triton.cdiv(n, BLOCK),)
+    grid = (batch_size * max_chunks_per_seq,)
```
[the remark this answers is not in the document]
Comment (wwens7) [reply mojo_opset/backends/ttx/kernels/ilu/kv_cache.py:265]: it's fine on ilu.

File: mojo_opset/backends/ttx/kernels/ilu/flash_attention.py [hunk 2]
```diff
@@ -241,10 +241,12 @@ def _paged_decode_gather_and_causal(
     """ILU paged decode: unpage KV + causal attention (one query token per batch row)."""
     batch_size, num_q_heads, head_dim = q.shape
     _, num_kv_heads, block_size, _ = key_cache.shape
-    o = torch.empty_like(q)
+    o = torch.zeros_like(q)
```
Comment (Neuromancer42) [inline mojo_opset/backends/ttx/kernels/ilu/flash_attention.py]: zeros_like会有额外的下发，是不是放在kernel内直接写入zeros更合适

File: mojo_opset/core/operators/attention.py [hunk 3]
```diff
@@ -9,6 +9,28 @@
...
 
+def assert_paged_prefill_contract(
+    cu_seqlens_q: torch.Tensor,
+    block_tables: torch.Tensor,
+    seqlens_kv: Optional[torch.Tensor],
+) -> None:
+    assert cu_seqlens_q.dtype == torch.int32
+    assert block_tables.dtype == torch.int32
+    q_lens = cu_seqlens_q[1:] - cu_seqlens_q[:-1]
+    kv_lens = q_lens if seqlens_kv is None else seqlens_kv
```
Comment (Neuromancer42) [inline mojo_opset/core/operators/attention.py]: 没有return？

**Review by zhangjihang-BD — DISMISSED** [review]

File: mojo_opset/backends/ttx/kernels/ilu/flash_attention.py (hunk 2 above)
Comment (wwens7) [reply mojo_opset/backends/ttx/kernels/ilu/flash_attention.py]: kernel 里不会写 padding 部分的值，似乎 empty 也没啥问题，这里 zeros 是想让输出尽量可控（padding == 0）

File: mojo_opset/core/operators/attention.py (hunk 3 above)
Comment (wwens7) [reply mojo_opset/core/operators/attention.py]: 我删一下，这两个本来是想跟 block table 做校验的

**Review by Neuromancer42 — APPROVED** [review]
LGTM
